src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/MoveQueryRepository.kt

```diff
@@ -212,8 +212,7 @@ class MoveQueryRepository(@Autowired val jdbcTemplate: JdbcTemplate) {
     offset: Long = 0
   ): List<Move> {
     val movesWithPersonAndJourneys = jdbcTemplate.query(
-      moveJourneySelectSQL +
-        "where m.supplier = ? and m.move_month = ? and m.move_year = ? and m.move_type = ? and m.drop_off_or_cancelled is not null " +
```

Comment:
This probably shouldn't have worked, but it did, due to a lack of whitespace before the 'where'.